src/pages/Trade/Swap/Swap.tsx

```diff
@@ -78,18 +80,23 @@ function Swap(props: propsIF) {
     // depending on location we pull data on the tx in progress differently
     const {
         tradeData: {
-            tokenA,
-            tokenB,
-            baseToken,
-            quoteToken,
-            isTokenAPrimary,
             primaryQuantity,
-            isDenomBase,
-            liquidityFee,
+
+            // liquidityFee
         },
     } = pathname.includes('/trade')
```

Comment:
Note: maybe we should double check that this doesn't break the existing functionality. Wasn't clear to me what should be different when using `useTradeData` vs `rtk`